### PR TITLE
AutoToggleClassOnScroll : Add multiple offset element feature, fix and unification

### PR DESCRIPTION
**Pull request by PascalPiche**
Allow multiple offset element.
Unify attribute name (rem- is now remove-).
Fix infiniteScrollAction name

### CODE REVIEW

File: src/modules/auto-toggle-class-on-scroll.js [hunk 1]
```diff
@@ -20,9 +20,9 @@
 	var ATTR_PREFIX = 'data-toggle-class-on-scroll-';
 
 	var ATTR_ADD_BEFORE = ATTR_PREFIX + 'add-before';
-	var ATTR_REM_BEFORE = ATTR_PREFIX + 'rem-before';
+	var ATTR_REM_BEFORE = ATTR_PREFIX + 'remove-before';
```
Comment (fhamon) [inline src/modules/auto-toggle-class-on-scroll.js]: We should change _rem_ to _remove_ in other modules too

Comment (nitriques) [inline src/modules/auto-toggle-class-on-scroll.js]: Good point @fhamon . @PascalPiche we need to choose one or the other and always use the same. I vote for 'rem'

File: src/modules/auto-toggle-class-on-scroll.js [hunk 2]
```diff
@@ -166,7 +168,7 @@
 				leave: leave
 			},
 			infiniteScroll: {
-				newListPage: refreshElementsList
+				pageLoaded: refreshElementsList
```
Comment (nitriques) [inline src/modules/auto-toggle-class-on-scroll.js]: This should have been another commit/PR

Comment (PascalPiche) [inline src/modules/auto-toggle-class-on-scroll.js]: dont have time to split all change in unique PR sorry. merge plz

File: src/modules/auto-toggle-class-on-scroll.js (hunk 1 above)
Comment (PascalPiche) [inline src/modules/auto-toggle-class-on-scroll.js]: Ya right.. will switch back to rem and make sure all other use the same. Will notify team member of the choice done.

**Pull request merged**